### PR TITLE
Clean up form package

### PR DESCRIPTION
**Pull request by wishdasher**
* Pre-bind callbacks
* Remove unnecessary lists from styles
* Manually fix weird prettier autoformatting

### CODE REVIEW

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit 8d9cf96b2a5689cdeecbd21853909d3467089d41

https://deploy-preview-284--wonder-blocks.netlify.com

**Comment by jeresig** [comment]
Deploy preview for *wonder-blocks* ready!

Built with commit e50d77a017e7fafeee04d71c57e4cc539efbe174

https://deploy-preview-284--wonder-blocks.netlify.com

File: packages/wonder-blocks-form/components/radio.md [hunk 1]
```diff
@@ -12,15 +12,14 @@ const styles = StyleSheet.create({
...
 
-<View style={[styles.row]}>
-    <Radio error={false} checked={false} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
-    <Radio error={false} checked={true} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
-    <Radio error={true} checked={false} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
-    <Radio error={true} checked={true} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
-    <Radio disabled={true} checked={false} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
-    <Radio disabled={true} checked={true} style={[styles.marginRight]} groupName={groupName} onChange={checked => handleChanged(checked)} />
+<View style={styles.row}>
+    <Radio error={false} checked={false} style={styles.marginRight} onChange={handleChange} />
```
Comment (kevinbarabash) [inline packages/wonder-blocks-form/components/radio.md]: Thanks for updating these too.  Having docs that promote best practices is awesome!

File: packages/wonder-blocks-form/components/radio-group.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ class RadioGroupPokemonExample extends React.Component {
         this.state = {
             selectedValue: null,
         };
+        this.handleChange = this.handleChange.bind(this);
```
Comment (kevinbarabash) [inline packages/wonder-blocks-form/components/radio-group.md]: I was always a little fuzzy as to whether you could use class properties when a constructor is present.

Comment (kevinbarabash) [reply packages/wonder-blocks-form/components/radio-group.md]: I'm guessing in this case styleguidist's parser doesn't allow class properties in general.

File: packages/wonder-blocks-form/components/choice-internal.js [hunk 3]
```diff
@@ -57,75 +57,82 @@ type Props = {|
...
-                        }
-                        onChange(!checked);
-                    }}
+                    onClick={this.handleClick}
                 >
                     {(state, handlers) => {
                         return (
                             <View
-                                style={[styles.wrapper]}
+                                style={styles.wrapper}
```
Comment (kevinbarabash) [inline packages/wonder-blocks-form/components/choice-internal.js]: nice

File: packages/wonder-blocks-form/components/choice-internal.js [hunk 4]
```diff
@@ -57,75 +57,82 @@ type Props = {|
...
+
     render() {
         const {
             label,
             description,
+            // eslint-disable-next-line no-unused-vars
             onChange,
             style,
+            // eslint-disable-next-line no-unused-vars
             variant,
```
Comment (kevinbarabash) [inline packages/wonder-blocks-form/components/choice-internal.js]: I see, they're in the pre-bound event handler now.  I wish there was a better way to exclude vars from a spread without having unused vars.

**Pull request merged**